### DISCUSSION TITLE
Exclude AnimatedPointSample from tests

### DISCUSSION THREAD

**Discussion by pauldendulk** [Mapsui/Mapsui#2075] [Development]
@inforithmics Do we have a mechanism to exclude samples from are tests? I see this in MapRegressionTests:
`public static object[] ExcludedSamples => _excludedSamples ??= new ISampleBase[]` but it is not used, is that intended for that purpose?

The AnimatedPointSample is flaky, which is to be expected because it is animated and thus the exact timing matters.

**Comment by pauldendulk** [comment]
We could also add our own [custom attribute](https://learn.microsoft.com/en-us/dotnet/standard/attributes/writing-custom-attributes) for this purpose.

**Comment by inforithmics** [comment]
Yes this is the intended usage, Because in the beginning a lot of samples wouldn't work. So I excluded it there an Attribute would be great Because we see directly in the code which samples are not regression tested. And Animated Point Sample is a Candidate because something dynamic is difficult to test with a static image.

The other solution is to Implement in the AnimatedPointSample the ISampleTest interface where I set dynamic Samples to specified state so that the static Image Creation works. For example the MutatingTriangleSample.
